pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -315,7 +346,7 @@ func newRunnerPodForCR(cr *litmuschaosv1alpha1.ChaosEngine, aUUID types.UID, aEx
 	labels := map[string]string{
 		"app": cr.Name,
 	}
-	podObj, err := pod.NewBuilder().
+	runnerPod, err := pod.NewBuilder().
```

Comment:
Could you please rebase to include newer changes to this file -- I suppose we have added the restart policy and ports info to the spec. I suppose this will also resolve the merge conflicts